Sources/MongoSwift/MongoClient.swift

```diff
@@ -655,6 +655,20 @@ public class MongoClient {
         return self.operationExecutor.execute(operation, client: self, session: session)
     }
 
+    /**
+     * Returns an `EventLoopBoundMongoClient`, a wrapper around this `MongoClient` that will use only the specified
+     * `EventLoop` for executing commands.
+     *
+     * - Parameters:
+     *   - eventLoop: An `EventLoop` which the returned `EventLoopBoundMongoClient` will be bound to.
+     *
+     * - Returns:
+     *    An `EventLoopBoundMongoClient` with the specified `EventLoop` bound to it.
```

Comment:
ditto flip ordering here: the client is bound to the event loop, not the other way around.